demo/app/app.component.ts

```diff
@@ -41,6 +43,11 @@ export class AppComponent implements AfterViewInit {
 
   ngAfterViewInit(): void {
     // example for getting the viewer by Id outside of the ac-map hierarchy
-    const viewer = this.viewersManager.getMap('main-map');
+    const map = this.mapsManagerService.getMap('main-map');
+    const viewer = map.getCesiumViewer();
+
+    const coordinateConverter = map.getCoordinateConverter();
+    this.draggableToMapService.init(coordinateConverter);
+    this.draggableToMapService.observable().subscribe(e => console.log(e));
```

Comment:
rename from observable to dragUpdates or somthing like this

Reply:
why? its the exposed observable - not like there are any other observables

Reply:
done